science/digital_rf/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           cxx11 1.1
+PortGroup           python 1.0
+
+github.setup        MITHaystack digital_rf 2.6.0
+maintainers         @MITHaystack {mit.edu:rvolz @ryanvolz} {mit.edu:swoboj @jswoboda}
+description         Read, write, and interact with data in the Digital RF and Digital Metadata formats.
+long_description    Tools to store RF and metadata for scientific engineering research projects.
+categories          science
+license             BSD
+platforms           darwin
+dist_subdir         digital_rf
+homepage            https://github.com/MITHaystack/digital_rf
+
+checksums           rmd160  f095ff1a80f7175f57a45048c826fe68b6dd56e9 \
+                    sha256  6468418a06c269964c9a5c5f145f7bfebf951226e3a85977226c7967fa50e691 \
+                    size    4528350
+
+configure.ldflags-delete -L${prefix}/lib
+
+# dependencies
+depends_lib-append port:py27-numpy \
+                   port:py27-matplotlib \
+                   port:py27-scipy \
+                   port:py27-pandas \
+                   port:py27-pkgconfig \
+                   port:hdf5 \
+                   port:py27-packaging \
+                   port:py27-dateutil \
+                   port:py27-h5py \
+                   port:py27-six \
+                   port:py27-tz \
+                   port:py27-watchdog \
+                   port:py27-mako \
+                   port:uhd \
+                   port:boost \
+                   port:gnutls \
+                   port:gnuradio
+
+
+# CMAKE configuration
+configure.args-append \
+    -DDRF_INSTALL_PREFIX_PYTHON=${python.prefix}\
+    -DPYTHON_EXECUTABLE=${frameworks_dir}/Python.framework/Versions/2.7/bin/python2.7\
+    -DDRF_DATA_PREFIX_PYTHON=${prefix} \
+    -DDRF_SCRIPT_PREFIX_PYTHON=${prefix}/bin
+
+use_configure \
+  yes
+
+build.target \
+  all
+build.cmd \
+  make
+destroot.cmd \
+  make
+destroot.target \
+  install
+
+destroot.destdir \
+  DESTDIR=${destroot}
```

Comment:
Again, this is already the default, so remove it, too.

Reply:
We're using the python portgroup to get the python prefix for everything but this seems to be causing issues with the install thus requiring us to do the all of the build commands. Should I remove the python portgroup and code in the prefix?

Reply:
Just check it and yes, the Python Portgroup is the issue with the install requiring those lines. It looks like any gnuradio related projects can't use this port group.

Reply:
If I understand correctly @jswoboda, you're saying that the Python portgroup doesn't play well with the CMake portgroup. CMake is the more important part since we were basically only using the Python portgroup to get the Python prefix, so I guess the cleanest solution is to just set that manually.